packages/utilities/psammead-locales/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-locales",
-  "version": "2.2.5",
+  "version": "2.2.6",
   "description": "A collection of locale configs, used in BBC World Service sites",
```

Comment:
Hi, you also need to update the version in the `psammead-locales/package-lock.json`. The Jenkins CI job is failing since package.json and package-lock.json files don't match.